lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -22,6 +22,9 @@ class Campaign {
   public $staff_pick;
   public $facts;
   public $solutions;
+  public $pre_step_header;
```

Comment:
The `pre_step` could probs be it's own item, and have a nested properties like you indicated in the description:

```
"pre_step": {
  "header": "Some header",
  "copy": {
    "raw": "lorem ipsum",
    "formatted": "<p>lorem ipsum</p>"
  }
}
```
something like that?